eos-update-flatpak-repos

```diff
@@ -709,12 +615,5 @@ if __name__ == '__main__':
     # ensure default remotes exist, such as eos-sdk and flathub
     _add_flatpak_repos()
 
-    # remove unused remotes, such as eos-external-apps and gnome-apps
-    _remove_remotes()
```

Comment:
Just checking: have you checked that all the code which is being deleted from `eos-update-flatpak-repos` has not been modified since the 3.4.0 release? I recall that we’ve been modifying it a fair amount recently.